src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
+jdbc.driver=com.mysql.jdbc.Driver
+jdbc.url=jdbc:mysql://localhost:3306/dev_db
+jdbc.username=root
+jdbc.password=root
 
+
+url.prefix = GET/weather-conditions" + "\\?" + "location=
+url.tomorrowURL = https://api.tomorrow.io/v4/timelines?location=
+url.apiKey = apikey=fvFU2JQBMP1QX7MeW44ghUQnNiy97uX6
```

Comment:
If you have time, consider checking how to persist sensitive information in spring. I'm sure this isn't that way :)

Reply:
I changed it :)